script/prerender/fileUtils.ts

```diff
@@ -51,7 +51,7 @@ export const createDiskCache = <K extends string, V>(prefix: string): Cache<K, V
 
 // Generates a release path for a file without a leading /, used when uploading the release to S3
 function prefixReleasePath(filepath: string) {
-  let basePath = assertDefined(process.env.PUBLIC_URL, 'PUBLIC_URL environment variable not set');
+  let basePath = `${PUBLIC_URL}`; // clone string to not leak the potential slice on next line.
```

Comment:
Should be fine without since slice() doesn't modify the original string.

Reply:
ah right